test/unit/utils/userStatus.test.js

```diff
@@ -0,0 +1,94 @@
+// Import the necessary dependencies and functions
+const chai = require("chai");
+const { expect } = chai;
+const { generateNewStatus, checkIfUserHasLiveTasks } = require("../../../utils/userStatus");
+
+describe("User Status Functions", function () {
+  describe("generateNewStatus", function () {
+    it('should return a new status object with state "ACTIVE" when isActive is true', function () {
+      const isActive = true;
+      const result = generateNewStatus(isActive);
+      expect(result.currentStatus.state).to.equal("ACTIVE");
```

Comment:
Can you please create the constants for all states? Or we can create a reference issue for this.

Reply:
We have the constants. will use it here.

Reply:
Done.